Documentation/guides/tracing.md

```diff
@@ -67,19 +61,33 @@ dbug: dotnet-dsrouter[0]
       Waiting for new ipc connection at endpoint "dotnet-diagnostic-dsrouter-21352".
 ```
 
-### For Android devices
+Use `dotnet-dsrouter android` instead, if you are testing on an
```

Comment:
The wording here should "mirror"/reflect the wording used earlier in the same section:

> For profiling an Android application running on an Android emulator:
> 
>     …
>
> For profiling an Android application running on an Android device:
>
>     $ adb reverse tcp:9000 tcp:9001
>     # `adb reverse` is required when using hardware devices
>    
>     $ dotnet-dsrouter android --verbose debug
>     …TODO: insert output here…

Probably related unanswered/unintuitive question: you say `adb reverse` is required, and if you use `adb reverse tcp:0 tcp:9001` it prints out the port number it randomly allocated.

*Where do you **use** this port number*?  Either `9000` or the randomly allocated port number; presumably that port number needs to be given to `dotnet-dsrouter android` *somehow*, right?

Reply:
The latest `dsrouter` prints out more information about ports. I updated what it prints, as it seems more explanatory.